server/app/services/applicant/question/EmailQuestion.java

```diff
@@ -46,7 +48,15 @@ public Optional<String> getEmailValue() {
       return emailValue;
     }
 
-    emailValue = applicantQuestion.getApplicantData().readString(getEmailPath());
+    ApplicantData applicantData = applicantQuestion.getApplicantData();
+    Optional<String> emailValue = applicantData.readString(getEmailPath());
+
+    if (emailValue.isEmpty()
+        && applicantQuestion
+            .getQuestionDefinition()
+            .containsPrimaryApplicantInfoTag(PrimaryApplicantInfoTag.APPLICANT_EMAIL)) {
+      emailValue = applicantData.getApplicant().getEmailAddress();
```

Comment:
Email is the only field that is not accessible via `applicantData`. All of the other fields go through `applicantData` to pull the value off of the applicant model.

Reply:
It is kinda weird to have only the one exception, so I would be in favor of adding a `getEmailAddress` to `applicantData`. Whether we should get it from `applicantData` or directly off of `applicant` going forward... I think that depends on how easy it would be to refactor to use `applicant` instead of `applicantData`. I remember when we looked at this before we decided we would need to keep `applicantData.getApplicant`, which would still be funky, but maybe there's not too many places we would need to do that.

Reply:
done.